app/views/alaveteli_pro/draft_info_request_batches/_draft_info_request_batch.html.erb

```diff
@@ -0,0 +1,39 @@
+<div id="draft-info-request-batch-<%= draft_info_request_batch.id %>"
+     class="request">
+  <h3 class="request__title">
+    <% if draft_info_request_batch.embargo_duration %>
+      <span class="embargo-indicator embargo-indicator--small">Private</span>
+    <% end %>
+    <span><%= _("Batch request") %></span>
```

Comment:
Note to self - there doesn't seem to be anything weird about the urls for embargoed requests, I think what must be happening is that `.title` returns an empty string for those ones, rather than nil.